### PR TITLE
Fr us 28

### LINKED ISSUE #31: User Story 28, User Profile displays Orders
As a registered user
When I visit my Profile Orders page, "/profile/orders"
I see every order I've made, which includes the following information:
- the ID of the order, which is a link to the order show page
- the date the order was made
- the date the order was last updated
- the current status of the order
- the total quantity of items in the order
- the grand total of all items for that order

### PR DESCRIPTION
**Pull request by fredrondina**
Completed US 28

Added /profile/orders index page

Added model method total_items to order 

closes #31

### CODE REVIEW

File: app/views/user/orders/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<h1>Your Orders</h1>
+<%@orders.each do |order| %>
+  <h4>Order ID: <%= link_to "#{order.id}", "/profile/orders/#{order.id}"%></h4>
```
Comment (brian-greeson) [inline app/views/user/orders/index.html.erb:3]: You could use a path prefix here and just pass in the object

File: spec/features/users/orders/index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# require "rails_helper"
+#
+RSpec.describe "As a user", type: :feature do
+    it "Displays every order ive made and its attributes" do
+      mike = Merchant.create(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
+      paper = mike.items.create(name: "Lined Paper", description: "Great for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 3)
+      pencil = mike.items.create(name: "Yellow Pencil", description: "You can write on paper with it!", price: 2, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 100)
+
+      user1 = User.new(
```
Comment (brian-greeson) [inline spec/features/users/orders/index_spec.rb:9]: User.create works too

**Review by brian-greeson — APPROVED** [review]
Looks good. Try using path prefixes more

**Review by kelshadarby — APPROVED** [review]
This all looks great! Nice work!

**Pull request merged**